README.md

```diff
@@ -1 +1,19 @@
-# hw01
\ No newline at end of file
+## Diana Reed
+
+### now
+Hi, I'm Diana Reed, a **MAPSS** student in Education and Society. A former Boston public school teacher, I'm now trying to "homeschool" my own two CPS kids, a 4th grader and an 8th grader, while finishing my thesis. We'll see how this goes!
+
+### then
+During other stages of my career, I've been an assessment developer, a project manager for large-scale assessments, and (briefly) a software engineer. Last year we all lived in Barcelona for half the school year, but we're now settled on the North Side of Chicago for the *foreseeable* future. 
+
+Here's a picture of me (with my daughter clumsily cropped out):
+
+![picture of me](/face.jpg)
+
+Some things we've baked during lockdown: 
+1. gingerbread 
+    1. https://cooking.nytimes.com/recipes/1012460-dark-molasses-gingerbread-with-whipped-cream would recommend. very moist and not too sweet. add some extra ginger and candied ginger
+1. easter egg bread    
+    1. https://www.tasteofhome.com/recipes/easter-egg-bread/ did not come out too well with gluten free flour. noticed too late that the "all-purpose flour" I was using was basically ground fava beans
```

Comment:
If you want to display the links, you can also use the footnote syntax:
easter egg brea[^1] 
[^1]: https://www.tasteofhome.com/....